Dockerfile

```
@@ -0,0 +1,34 @@
+FROM ubuntu:18.04
+
+RUN apt-get update
+RUN apt-get upgrade
+
+RUN apt-get install -y build-essential wget
+RUN apt-get install -y python3 python3-pip python3-dev python3-venv
+RUN apt install parallel -y
+
+ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.2.1/wait /wait
+RUN chmod +x /wait
+
+RUN mkdir /colouring-london
+COPY app /colouring-london/app
+COPY migrations /colouring-london/migrations
+COPY etl /colouring-london/etl
+
+ENV NODE_VERSION=16.13.2
+RUN apt install -y curl
```

```suggestion
RUN apt-get install -y curl
```

This prevents some warning messages from appearing in logs